src/main/scala/org/broadinstitute/hail/variant/VariantSampleMatrix.scala

```diff
@@ -706,17 +703,18 @@ class VariantSampleMatrix[T](val metadata: VariantMetadata,
     val localSampleAnnotationsBc = sampleAnnotationsBc
 
     copy(vaSignature = newVAS,
-      rdd = rdd.mapValuesWithKey { case (v, (va, gs)) =>
-        val serializer = SparkEnv.get.serializer.newInstance()
-        val zeroValue = serializer.deserialize[U](ByteBuffer.wrap(zeroArray))
-
-        (mapOp(va, gs.iterator
-          .zip(localSampleIdsBc.value.iterator
-            .zip(localSampleAnnotationsBc.value.iterator)).foldLeft(zeroValue) {
-          case (acc, (g, (s, sa))) =>
-            seqOp(acc, v, va, s, sa, g)
-        }), gs)
-      }.asOrderedRDD[Locus])
+      rdd = rdd.mapValuesWithKey {
+        case (v, (va, gs)) =>
```

Comment:
Unnecessary change in indentation (I think?).  I've been seeing IntelliJ do this sporadically and I'm not sure what's up with that.